Lombiq.Tests.UI/Models/IFailureDumpItem.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace Lombiq.Tests.UI.Models;
+
+/// <summary>
+/// Represents a failure dump item.
+/// </summary>
+public interface IFailureDumpItem : IDisposable
+{
+    /// <summary>
+    /// Return a <see cref="Stream"/> associated to failure dump item.
```

Comment:
Also elaborate the `<returns>` similarly.

```suggestion
    /// Gets the <see cref="Stream"/> associated to the failure dump item.
```